packages/expo-localization/android/src/main/java/expo/modules/localization/LocalizationModule.kt

```diff
@@ -0,0 +1,83 @@
+package expo.modules.localization
+
+import android.content.Context
+import expo.modules.core.ExportedModule
+import android.os.Bundle
+import expo.modules.core.interfaces.ExpoMethod
+import android.text.TextUtils
+import android.os.Build.VERSION
+import android.os.Build.VERSION_CODES
+import android.view.View
+import expo.modules.core.Promise
+import java.lang.ref.WeakReference
+import java.text.DecimalFormatSymbols
+import java.util.*
+import kotlin.collections.ArrayList
+
+class LocalizationModule(context: Context) : ExportedModule(context) {
+  private val contextRef: WeakReference<Context> = WeakReference(context)
+
+  private val applicationContext: Context?
+    get() = contextRef.get()?.applicationContext
+
+  override fun getName() = "ExpoLocalization"
+
+  override fun getConstants(): Map<String, Any> {
+    val constants = HashMap<String, Any>()
+    val bundle = bundledConstants
```

Comment:
Here's my suggestion, but it is up to you
```suggestion
  override fun getConstants(): Map<String, Any> {
    val constants = hashMapOf<String, Any>()
    val bundle = bundledConstants
```

or maybe

```suggestion
  override fun getConstants(): Map<String, Any> {
    val constants = mutableMapOf<String, Any>()
    val bundle = bundledConstants
```